lib/PHPUnitTagProvider.coffee

```diff
@@ -0,0 +1,69 @@
+DocblockTagProvider = require "./DocblockTagProvider"
+
+module.exports =
+
+##*
+# Provides autocompletion for PHP-unit annotation tags.
+##
+class PHPUnitTagProvider extends DocblockTagProvider
+    ###*
+     * @inheritdoc
+    ###
+    addSuggestions: (tagList, prefix) ->
+        suggestions = []
+
+        for tag in tagList
+            # TODO: This URL should go to the config file
+            documentationUrl = 'https://phpunit.de/manual/current/en/appendixes.annotations.html'
```

Comment:
Could you complete the TODO and move this to the configuration class (you can take a gander at `DocblockTagProvider`, which also fetches it from the config).